lib/matplotlib/axes/_axes.py

```diff
@@ -197,6 +198,11 @@ def set_xlabel(self, xlabel, fontdict=None, labelpad=None, **kwargs):
         xlabel : str
             The label text.
 
+        loc : str, default: None
+            Choose label position {'left'|'center'|'right'}. None defaults to
+            rcParam 'xaxis.titlelocation'. Equivalent to parsing 'x' and
+            'horizontalalignment' kwargs.
```

Comment:
Documentation style:
~~~
        loc : {'left', 'center', 'right'}, default: :rc:`xaxis.titlelocation`
            The label position. This is a high-level alternative for passing parameters *x* and
            *horizonatalalignment* and overwrites these settings.
~~~

(See https://matplotlib.org/devdocs/devel/documenting_mpl.html#writing-docstrings for our documentation standard.)

Note: The last sentence does not reflect current behavior, but is what I propose below; so maybe subject to discussion.